src/components/HeaderForm.js

```diff
@@ -527,6 +533,20 @@ class HeaderForm extends Component {
     if (tracks) {
       this.setState({ tracks: tracks });
       console.log("New tracks have been population");
+    } else if (isValidURL(this.state.bedFile)){
+      // Try to retrieve tracks if the current bed file is an URL
```

Comment:
It looks like we're assuming that, if the BED file is not a URL and is actually mounted on the server, it also doesn't reference any chunks by URL and the server will always have sent us full track data for every region.

I don't think we should make that assumption on the client; we should be able to ask for the tracks for a BED region whenever we don't have them.